src/codegen/functions/function.py

```diff
@@ -100,3 +106,12 @@ def output_parameters(self):
     def adaptor_parameter(self):
         """List of adaptor parameters: The list of adaptor parameters in the function."""
         return self._adaptor_parameter
+
+    @property
+    def base_parameters(self):
+        """List of all parameters: The list of all in the function."""
+        return self._base_parameters
+    
+    @property
+    def stream_response(self):
+        return self._stream_response
```

Comment:
This property is missing a docstring. stream_response means that the corresponding gRPC method uses server streaming to send multiple responses.

Reply:
Added docstring to stream_response param